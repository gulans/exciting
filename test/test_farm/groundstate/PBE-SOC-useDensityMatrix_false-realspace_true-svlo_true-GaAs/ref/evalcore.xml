<?xml version="1.0" encoding="UTF-8"?>
<evalcore>
  <species is="1" chemicalSymbol="Ga">
    <atom ia="1">
      <state ist="1" n="1" l="0" k="0" eigenvalue="-3.754585959548e2"/>
      <state ist="2" n="2" l="0" k="0" eigenvalue="-4.598297950612e1"/>
      <state ist="3" n="2" l="1" k="1" eigenvalue="-4.066003495875e1"/>
      <state ist="4" n="2" l="1" k="1" eigenvalue="-3.964371717903e1"/>
      <state ist="5" n="3" l="0" k="0" eigenvalue="-5.086464445026e0"/>
      <state ist="6" n="3" l="1" k="1" eigenvalue="-3.367684258131e0"/>
      <state ist="7" n="3" l="1" k="1" eigenvalue="-3.235294982276e0"/>
    </atom>
  </species>
  <species is="2" chemicalSymbol="As">
    <atom ia="1">
      <state ist="1" n="1" l="0" k="0" eigenvalue="-4.300133686654e2"/>
      <state ist="2" n="2" l="0" k="0" eigenvalue="-5.408247731600e1"/>
      <state ist="3" n="2" l="1" k="1" eigenvalue="-4.825974759202e1"/>
      <state ist="4" n="2" l="1" k="1" eigenvalue="-4.691622485954e1"/>
      <state ist="5" n="3" l="0" k="0" eigenvalue="-6.481349248214e0"/>
      <state ist="6" n="3" l="1" k="1" eigenvalue="-4.526332794514e0"/>
      <state ist="7" n="3" l="1" k="1" eigenvalue="-4.342362616850e0"/>
    </atom>
  </species>
</evalcore>
