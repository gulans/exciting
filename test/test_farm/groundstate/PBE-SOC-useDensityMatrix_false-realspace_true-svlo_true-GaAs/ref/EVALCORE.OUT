 
Species :    1 (Ga), atom :    1
 n =  1, l =  0, k =  1 :   -375.4585960    
 n =  2, l =  0, k =  1 :   -45.98297951    
 n =  2, l =  1, k =  1 :   -40.66003496    
 n =  2, l =  1, k =  2 :   -39.64371718    
 n =  3, l =  0, k =  1 :   -5.086464445    
 n =  3, l =  1, k =  1 :   -3.367684258    
 n =  3, l =  1, k =  2 :   -3.235294982    
 
Species :    2 (As), atom :    1
 n =  1, l =  0, k =  1 :   -430.0133687    
 n =  2, l =  0, k =  1 :   -54.08247732    
 n =  2, l =  1, k =  1 :   -48.25974759    
 n =  2, l =  1, k =  2 :   -46.91622486    
 n =  3, l =  0, k =  1 :   -6.481349248    
 n =  3, l =  1, k =  1 :   -4.526332795    
 n =  3, l =  1, k =  2 :   -4.342362617    
